src/WebJobs.Script.WebHost/Security/KeyManagement/DefaultKeyValueConverterFactory.cs

```diff
@@ -24,7 +24,7 @@ public DefaultKeyValueConverterFactory(ScriptSettingsManager settingsManager)
         }
 
         // In Linux Containers AzureWebsiteLocalEncryptionKey will be set, enabling encryption
-        private static bool IsEncryptionSupported() => _settingsManager.IsAppServiceEnvironment || _settingsManager.GetSetting(AzureWebsiteLocalEncryptionKey) != null;
+        private static bool IsEncryptionSupported() => _settingsManager.IsAppServiceEnvironment;
```

Comment:
Instead, change the function to the below, to make it Seabreeze specific (so these changes don't affect Linux dedicated) for example.

```csharp
private static bool IsEncryptionSupported()
{
    if (_settingsManager.IsLinuxContainerEnvironment)
    {
        // TEMP: https://github.com/Azure/azure-functions-host/issues/3035
        return false;
    }

    return _settingsManager.IsAppServiceEnvironment || _settingsManager.GetSetting(AzureWebsiteLocalEncryptionKey) != null;
}
```

Reply:
Also remove that old "In Linux Containers" comment I had on the method :)

Reply:
Done.